ref/api.md

```diff
@@ -506,6 +507,38 @@ Bugs and limitations:
     `hasTrailingZero` above, and
     [#178](https://github.com/scottlamb/moonfire-nvr/issues/178).
 
+### `DELETE /api/cameras/<uuid>/<stream>/view.mp4`
```

Comment:
> While `DELETE /api/cameras/.../.../view.mp4` is a little strange, given `GET view.mp4` does a live transformation to MP4 to justify its name, I do like the `DELETE` endpoint is guessable from the `GET` endpoint.

Yeah, that name is a bit strange. The guessability is nice but hopefully not too important when we have these API docs.

My first thought is that maybe this should be a `POST` to `/recordings`. We also should take in the `csrf` parameter for this as we do for other non-`GET` stuff, and that gives us a place to put it in a similar format, and to attach more preconditions and such if desired. What do you think?

Reply:
`csrf` is a good callout. Will update.

Is there a sub-path of `POST /api/camera/.../.../recordings/XYZ` that could work? Or `POST /api/camera/.../.../XYZ`?